cockroachcloud/cloud-api.md

```diff
@@ -279,7 +279,7 @@ Where:
 
 ## Get information about a cluster's nodes
 
-To retrieve information about a cluster's nodes, including the node status, make a `GET` request to the `/v1/clusters/{cluster_id}/nodes` endpoint. The service account associated with the secret key must have `ADMIN` or `READ` [permission](console-access-management.html#service-accounts) to retrieve information about an organization's clusters.
+To retrieve information about a cluster's nodes, including the node status, make a `GET` request to the `/v1/clusters/{cluster_id}/nodes` endpoint. The service account associated with the secret key must have `ADMIN` or `READ` [permission](managing-access.html#manage-service-accounts) to retrieve information about an organization's clusters.
```

Comment:
In the new auth model, this can be done by Cluster Admin or Cluster Developer roles.